src/app/components-small/template-tab-form/template-tab-form.component.ts

```diff
@@ -0,0 +1,53 @@
+import { Component, OnInit, Input, Output, SimpleChanges, EventEmitter } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+
+@Component({
+    selector: 'qhana-template-tab-form',
+    templateUrl: './template-tab-form.component.html',
+    styleUrls: ['./template-tab-form.component.sass']
+})
+export class TemplateTabFormComponent implements OnInit {
+    @Input() formValues: {
+        name: string | null,
+        description: string | null,
+        sortKey: number | null,
+        filterString: string | null
+    } = {
+        name: '',
+        description: '',
+        sortKey: 0,
+        filterString: '{}'
+    };
+
+    @Output() formSubmit = new EventEmitter<{
```

Comment:
This type should be an exported interface, as the event output is expected to be used outside, but the type cannot be reused like this.